DotaStats/Modules/MatchInfoModule/View/AdditionalMatchInfoTableViewCell.swift

```diff
@@ -0,0 +1,105 @@
+import UIKit
+
+class AdditionalMatchInfoTableViewCell: UITableViewCell {
+    static let reuseIdentifier = "AdditionalMatchInfoTableViewCell"
+    var matchID = String()
+    
+    private let regionStackView = UIStackView()
+    private let skillStackView = UIStackView()
+
+    private let matchIdCopyButton = CopyButton()
+    private let regionNameLabel = UILabel()
+    private let regionLabel = UILabel()
+    private let skillNameLabel = UILabel()
+    private let skillLabel = UILabel()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setup()
+        setUpConstraints()
```

Comment:
Давай в едином стиле напишем?
```suggestion
        setupConstraints()
```